pyaedt/edb_core/edb_data/siwave_simulation_setup_data.py

```diff
@@ -0,0 +1,897 @@
+        self.sim_setup_info.SimulationSettings.AdvancedSettings.InfGndLocation = value
+        self._parent._update_setup()
+
+    @max_coupled_lines.setter
+    def max_coupled_lines(self, value):
+        self.sim_setup_info.SimulationSettings.UseCustomSettings = True
+        self.sim_setup_info.SimulationSettings.AdvancedSettings.MaxCoupledLines = value
+        self._parent._update_setup()
+
+    @automatic_mesh.setter
+    def automatic_mesh(self, value):
+        self.sim_setup_info.SimulationSettings.UseCustomSettings = True
+        self.sim_setup_info.SimulationSettings.AdvancedSettings.MeshAutoMatic = value
+        self._parent._update_setup()
+
+    @perform_erc.setter
+    def perform_erc(self, value):
+        self.sim_setup_info.SimulationSettings.AdvancedSettings.PerformERC = value
+        self._parent._update_setup()
+
+    @mesh_frequency.setter
+    def mesh_frequency(self, value):
+        self.sim_setup_info.SimulationSettings.UseCustomSettings = True
+        self.sim_setup_info.SimulationSettings.AdvancedSettings.MeshFrequency = value
+        self._parent._update_setup()
+
+
+class SiwaveDCAdvancedSettings(object):
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def sim_setup_info(self):
+        """Edb Internal Simulation Setup Object."""
+
+        return self._parent._edb_sim_setup_info
+
+    @property
+    def min_void_area(self):
+        """The minimum area below which voids are ignored.
+
+        Returns
+        -------
+        str
+        """
+        return self.sim_setup_info.SimulationSettings.DCAdvancedSettings.DcMinVoidAreaToMesh
+
+    @property
+    def min_plane_area(self):
+        """The minimum area below which geometry is ignored.
+
+        Returns
+        -------
+        str
+        """
+        return self.sim_setup_info.SimulationSettings.DCAdvancedSettings.DcMinPlaneAreaToMesh
+
+    @property
+    def energy_error(self):
+        """Energy error.
+
+        Returns
+        -------
+        str
+        """
+        return self.sim_setup_info.SimulationSettings.DCAdvancedSettings.EnergyError
+
+    @property
+    def max_init_mesh_edge_length(self):
+        """Initial mesh maximum edge length.
+
+        Returns
+        -------
+        str
+        """
+        return self.sim_setup_info.SimulationSettings.DCAdvancedSettings.MaxInitMeshEdgeLength
+
+    @property
+    def max_num_pass(self):
+        """Maximum number of passes.
+
+        Returns
+        -------
+        int
+        """
+        return self.sim_setup_info.SimulationSettings.DCAdvancedSettings.MaxNumPasses
+
+    @property
+    def min_num_pass(self):
+        """Minimum number of passes.
+
+        Returns
+        -------
+        int
+        """
+        return self.sim_setup_info.SimulationSettings.DCAdvancedSettings.MinNumPasses
+
+    @property
+    def mesh_bondwires(self):
+        """Mesh bondwires.
```

Comment:
This property and several that follow show a Boolean as a return value. If Boolean is correct, the description should be formatted as "Whether to...". For this property, the description should be "Whether to mesh boundaries."